caravel/assets/javascripts/SqlLab/components/SqlEditorLeft.jsx

```diff
@@ -73,11 +77,17 @@ class SqlEditorTopToolbar extends React.Component {
   }
   fetchDatabaseOptions() {
     this.setState({ databaseLoading: true });
-    const url = '/databaseasync/api/read';
+    const url = '/databaseasync/api/read?_flt_0_expose_in_sqllab=1';
```

Comment:
it's part of the FAB rest mini language, not sure if we need to explain it in this context (and all other contexes where it's used)